server/test/services/migration/ProgramMigrationServiceTest.java

```diff
@@ -107,98 +112,24 @@ public void deserialize_notEnoughInfoToCreateProgramDef_returnsError() {
 
   @Test
   public void deserialize_jsonHasAllInfo_returnsResult() {
-    ErrorAnd<ProgramMigrationWrapper, String> result = service.deserialize(EXAMPLE_PROGRAM_JSON);
+    ErrorAnd<ProgramMigrationWrapper, String> result =
+        service.deserialize(PROGRAM_JSON_WITH_ONE_QUESTION);
 
     assertThat(result.isError()).isFalse();
     ProgramMigrationWrapper wrapperResult = result.getResult();
     assertThat(wrapperResult.getProgram()).isNotNull();
+
     ProgramDefinition program = wrapperResult.getProgram();
-    assertThat(program.adminName()).isEqualTo("import-program-sample");
+    QuestionDefinition question = wrapperResult.getQuestions().get(0);
+
+    assertThat(program.adminName()).isEqualTo("minimal-sample-program");
     assertThat(program.adminDescription()).isEqualTo("desc");
     assertThat(program.externalLink()).isEqualTo("https://github.com/civiform/civiform");
     assertThat(program.displayMode()).isEqualTo(DisplayMode.PUBLIC);
     assertThat(program.programType()).isEqualTo(ProgramType.DEFAULT);
+    assertThat(question.getName()).isEqualTo("Name");
+    assertThat(question.getDescription()).isEqualTo("The applicant's name");
+    assertThat(question.getQuestionText().getDefault())
+        .isEqualTo("Please enter your first and last name");
   }
-
-  /**
-   * This contains the bare minimum needed to parse JSON into a program definition. The
-   * admin_program_migration.test.ts browser test has tests for a program with many blocks and
-   * questions.
-   */
-  public static final String EXAMPLE_PROGRAM_JSON =
```

Comment:
I moved this to `AdminImportControllerTest` so it lives with all the other sample json strings I created for testing.